projects/exercises/01/drive_in_line.cpp

```diff
@@ -0,0 +1,13 @@
+#include <STSL/RJRobot.h>
+
+using namespace std;
+
+int main()
+{
+    RJRobot robot(RobotType::REAL); //Initialize a RJ Robot that controls a real robot
+    
+    robot.setDriveMotors(127, 127); //Drive both robot motors forward at max speed
```

Comment:
```suggestion
    robot.setDriveMotors(1.0, 1.0); //Drive both robot motors forward at max speed
```

Drive motor valid ranges are -1.0 to 1.0